public/usage-examples/graphics/has_font/has_font-1-simple.py

```diff
@@ -0,0 +1,19 @@
+from splashkit import *
+
+open_window("Has Font", 800, 600)
+clear_screen()
+
+# Check if the font exists
+font_check = has_font("NORMAL_FONT")
+
+# Display the text to show the result
+if font_check:
+    draw_text("Font found!", color_black(), 300, 300)
```

Comment:
for this `draw_text` function the python signature is `def draw_text(text, clr, fnt, font_size, x, y):`
I think you mean to use this overload of the function instead `def draw_text_no_font_no_size(text, clr, x, y):`

To fix this simply replace the function call on line 11 and 13 with `draw_text_no_font_no_size()` instead of `draw_text()`